.github/workflows/py-run-tests.yml

```diff
@@ -30,12 +40,7 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@2bd53f9a4d1dd1cd21eaffcc01a7b91a8e73ea4c
       with:
-        python-version: '3.x'
-
-    - name: Create and activate virtual environment
-      run: |
-        python -m venv .venv
-        source .venv/bin/activate
```

Comment:
Tests in GH actions are independent, so there is no need to activate a venv using a specific Python config.